# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
# Move a robot arm
#
# goal message = the request
int32 arm				# number of the arm to control (1=left, 2=right)
int32 LEFTARM=1
int32 RIGHTARM=2

geometry_msgs/PoseStamped goal_position		# the target position of the wrist of the robot arm

